clang/test/Driver/sycl-offload-new-driver.c

```diff
@@ -45,7 +45,7 @@
 // RUN:          -Xspirv-translator -translator-opt -### %s 2>&1 \
 // RUN:   | FileCheck -check-prefix WRAPPER_OPTIONS_TRANSLATOR %s
 // WRAPPER_OPTIONS_TRANSLATOR: clang-linker-wrapper{{.*}} "--triple=spir64"
-// WRAPPER_OPTIONS_TRANSLATOR-SAME: "--llvm-spirv-options=-translator-opt"
+// WRAPPER_OPTIONS_TRANSLATOR-SAME: "--llvm-spirv-options={{.*}}"
```

Comment:
The check for `translator-opt` is lost here with the wildcard usage.  Can you add it back, with a trailing wildcard?

Reply:
Done. Thanks